### PR TITLE
Restrict google-cloud-texttospeach to <v2

### PR DESCRIPTION
**Pull request by ashb**
Version 2 of this library causes the following pylint errors:

> airflow/providers/google/cloud/hooks/text_to_speech.py:59:27: E1123: Unexpected keyword argument 'client_info' in constructor call (unexpected-keyword-arg)
> airflow/providers/google/cloud/hooks/text_to_speech.py:99:15: E1123: Unexpected keyword argument 'input_' in method call (unexpected-keyword-arg)

This was noticed when I tried to update requirements in #9136 (now rebased to be on top of this PR)

---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'google-cloud-speech>=0.36.3',
     'google-cloud-storage>=1.16',
     'google-cloud-tasks>=1.2.1',
-    'google-cloud-texttospeech>=0.4.0',
+    'google-cloud-texttospeech>=0.4.0<2',
```
Comment (ashb) [inline setup.py]: Bah, typo, missing a comma here

**Review by mik-laj — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

**Comment by ashb** [comment]
Looks like there might be a single slow/wedged test. 38m and counting :(

**Comment by ashb** [comment]
Though it's passed in https://github.com/apache/airflow/pull/9136 which includes the exact same commit. I'm going to merge anyway

**Comment by ashb** [comment]
I've set this for 1.10.11 -- I'm not sure if we have this dep in 1.10.x or not, but we need to update that version even with backporting providers..

**Pull request merged**